third_party/spdk/BUILD

```diff
@@ -5,5 +5,6 @@ exports_files(["LICENSE"])
 cc_library(
   name = "nvme_lib",
   hdrs = ["nvme.h"],
+  deps = ["@com_google_absl//absl/base"],
```

Comment:
intentional?

Reply:
yeah, nvme needs the absl dependency for the ABSL_ATTRIBUTE_PACKED annotation. It seems like before the nvme lib was always linked with the scsi lib so it somehow got the absl dependency from the scsi dependency chain? But when you try to just use the nvme lib without the scsi lib, then it breaks.